containers/hydra/postbuild.py

```diff
@@ -59,8 +59,9 @@ def create_signature(path: str) -> str:
     returns nix store path of the signature
     """
 
-    result = subprocess.run(["/setup/sign.sh", path],
-                            stdout=subprocess.PIPE, check=False)
+    result = subprocess.run(
```

Comment:
Irrelevant styling changes. 
Even if new styling was preferred, style changes should be in a separate commit. 
Here I don't see much use for new styling anyway.